src/assets/manifest.ts

```diff
@@ -54,3 +37,27 @@ export async function writeWebManifest(
     "utf-8",
   );
 }
+
+export async function overrideWebManifestIcons(manifestFile: string) {
+  const manifest = await readWebManifestFile(manifestFile);
+
+  return !!manifest?.manifest && !("icons" in manifest.manifest);
+}
+
+function resolveWebManifestFile(ctx: QwikPWAContext) {
+  return resolve(ctx.publicDir, ctx.options.webManifestFilename);
+}
+
+async function readWebManifestFile(manifestFile: string) {
+  const isFile = await lstat(manifestFile)
+    .then((stat) => stat.isFile())
+    .catch(() => false);
+  if (!isFile) return;
+
+  return {
+    manifestFile,
```

Comment:
we already have the manifestFile by passing it, so let's remove this line too.

Reply:
done